emails/migrations/0061_profile_locale.py

```diff
@@ -0,0 +1,21 @@
+# Generated by Django 4.2.10 on 2024-03-14 21:51
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
```

Comment:
It would be nice if we can populate the old profile with existing data in `extra_data`. Here is an example of SQL command I used to set default value:
https://github.com/mozilla/fx-private-relay/blob/main/emails/migrations/0058_profile_onboarding_free_state.py#L19-L22

Here's the documentation I created to generate the PSQL and SQLite queries: https://github.com/mozilla/fx-private-relay/pull/4124/files

Here's how to convert Django Queryset into SQL queries: https://stackoverflow.com/questions/3748295/getting-the-sql-from-a-django-queryset